enterprise-suite/templates/es-grafana-deployment.yaml

```diff
@@ -19,6 +19,9 @@ spec:
       labels:
         app: grafana
         component: server
+      annotations:
```

Comment:
Do we need any new label_configs for these?

Reply:
regular pod rules seem to be workin fine